src/core/data-cite.js

```diff
@@ -52,12 +52,17 @@ function requestLookup(conf) {
     if (frag) {
       href = new URL(frag, href).href;
     }
+    const cite = document.createElement("cite");
     switch (elem.localName) {
       case "a": {
         if (elem.textContent === "") {
           elem.textContent = title;
         }
         elem.href = href;
+        // ToDo (It creates problem because HTML does not know ehere to place newly created node that is <cite> <a..> </a> </cite>)
```

Comment:
@marcoscaceres I was able to add `<cite>` for dfn successfully but it creates problem with `<a>` probably because HTML does not know where to place newly created node <cite>. 
In dfn we are able to tackle this by appending `<cite>` to dfn element. I tried looking for solution but could not figure out any concrete solution. Do you have any suggestions?

Reply:
> because HTML does not know where to place newly created node 

Not sure what you mean here? I'd suggest stepping into  wrapInner(cite, elem); and seeing exactly what's going on.

Reply:
The below line just appends `elem` into `cite` and nothing more. You should instead do `elem.replaceWith(cite); cite.append(elem);`.

Reply:
@saschanaz That worked. Thanks. 😄 
I ran into some issues recently while I was trying an alternative fix. But that was quick. 😬